### PR TITLE
Feature/environment update

### PR DESCRIPTION
**Pull request by el-hoshino**

### CODE REVIEW

**Comment by el-hoshino-xcode-cloud** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Diff Volume Check: There's too much diff. Please make PRs smaller.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Failed to get coverage from file:///Volumes/workspace/resultbundle.xcresult/</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Danger/Dangerfile.swift#L13** - Comment Spacing Violation: Prefer at least one space after slashes for comments. (comment_spacing)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Danger/Dangerfile.swift#L39** - Comment Spacing Violation: Prefer at least one space after slashes for comments. (comment_spacing)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **ld#L1** - dylib (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/lib_InternalSwiftSyntaxParser.dylib) was built for newer macOS version (10.14.6) than being linked (10.13)
  </td>
    </tr>
  </tbody>
</table>


Checking Item | Result
| ---| --- |
Base Branch Check | :tada:
Merge Commit Non-Existence Check | :tada:
Diff Volume Check | :thinking:
ChangeLog Modification Check | :tada:

## Feature PR Check
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against c70a6ef492ad20814a19b23a435cc16b9361ae2f
</p>

File: AppPackage/Package.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+let package = Package(
+    name: "AppPackage",
+    platforms: [
+        .iOS(.v14),
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "AppPackage",
+            targets: ["AppPackage"]),
```
Comment (el-hoshino-xcode-cloud) [inline AppPackage/Package.swift:15]: - :warning: Multiline arguments should have their surrounding brackets in a new line. (`multiline_arguments_brackets`)

File: AppPackage/Package.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/nalexn/ViewInspector", from: "0.9.3"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "AppPackage",
+            dependencies: [
+            ]),
```
Comment (el-hoshino-xcode-cloud) [inline AppPackage/Package.swift:28]: - :warning: Multiline arguments should have their surrounding brackets in a new line. (`multiline_arguments_brackets`)

File: AppPackage/Package.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        .testTarget(
+            name: "AppPackageTests",
+            dependencies: [
+                "AppPackage",
+                "ViewInspector",
+            ]),
```
Comment (el-hoshino-xcode-cloud) [inline AppPackage/Package.swift:34]: - :warning: Multiline arguments should have their surrounding brackets in a new line. (`multiline_arguments_brackets`)

File: Danger/Dangerfile.swift [hunk 4]
```diff
@@ -1,482 +1,17 @@
...
-    
-}
 
+// MARK: - 便利な拡張
 private extension ProcessInfo {
     
     static var xcTestResultPath: String {
         
         // If running on Xcode Cloud it should be able to get path from `$CI_RESULT_BUNDLE_PATH` variable
         processInfo.environment["CI_RESULT_BUNDLE_PATH"] ??
```
Comment (el-hoshino-xcode-cloud) [inline Danger/Dangerfile.swift:13]: - :warning: Assigning default value from an optional variable may implicitly hide unintended `nil` values at development. Please make sure this is what you intended to do, and in that case please temporarily disable this rule at current line. (`optional_default_value`)

File: Danger/Dangerfile.swift [hunk 5]
```diff
@@ -493,47 +28,18 @@ private extension SwiftLint.SwiftlintPath {
...
+// SwiftLint のワーニング等確認
 SwiftLint.lint(.modifiedAndCreatedFiles(directory: nil), inline: true, configFile: "swiftlint.yml", swiftlintPath: .installedBySwiftPM)
 
-// Xcode summary warnings check.
+// Xcode ビルドのワーニング等確認
 danger.kantoku.parseXCResultFile(at: ProcessInfo.xcTestResultPath, configuration: .default)
 
 // Xcode test coverage check.
 // temporarily disable this because hasn't found a way to get coverage at post_xcodebuild yet
 //Coverage.xcodeBuildCoverage(.xcresultBundle(ProcessInfo.xcTestResultPath), minimumCoverage: 60)
```
Comment (el-hoshino-xcode-cloud) [inline Danger/Dangerfile.swift:42]: - :warning: Prefer at least one space after slashes for comments. (`comment_spacing`)

File: AppPackage/Tests/AppPackageTests/QRPictoreGeneratorTests.swift [hunk 6]
```diff
@@ -28,3 +28,13 @@ class QRPictoreGeneratorTests: XCTestCase {
     }
     
 }
+
+private extension QRPictoreGeneratorTests {
+    
+    // swiftlint result reporting check
+    func dummy(
+        aaa: Int,
+        bbb: Int) {
```
Comment (el-hoshino) [inline AppPackage/Tests/AppPackageTests/QRPictoreGeneratorTests.swift:37]: aaa

**Pull request merged**